integration_test/src/test_runner/mod.rs

```diff
@@ -0,0 +1,327 @@
+                Ok(odbc_sys::SQLGetTypeInfo(statement.as_sys(), data_type))
+            }
+        }
+        "sqltables" => {
+            check_array_length(function, 9);
+            unsafe {
+                Ok(odbc_sys::SQLTables(
+                    statement.as_sys(),
+                    str_or_null(&function[1]),
+                    to_i16(&function[2]),
+                    str_or_null(&function[3]),
+                    to_i16(&function[4]),
+                    str_or_null(&function[5]),
+                    to_i16(&function[6]),
+                    str_or_null(&function[7]),
+                    to_i16(&function[8]),
+                ))
+            }
+        }
+        /*
+        "sqlprimarykeys" => {
+            unsafe {
+                Ok(odbc_sys::SQLPrimaryKeys(
+                    statement.as_sys(),
+                    str_or_null(&function[1]),
+                    to_i16(&function[2]),
+                    str_or_null(&function[3]),
+                    to_i16(&function[4]),
+                    str_or_null(&function[5]),
+                    to_i16(&function[6]),
+                ))
+            }
+        }
+        "sqlspecialcolumns" => {
+            unsafe {
+                Ok(odbc_sys::SQLSpecialColumns(
+                    statement.as_sys(),
+                    to_i16(&function[1]),
+                    str_or_null(&function[2]),
+                    to_i16(&function[3]),
+                    str_or_null(&function[4]),
+                    to_i16(&function[5]),
+                    str_or_null(&function[6]),
+                    to_i16(&function[7]),
+                    to_i16(&function[8]),
+                    to_i16(&function[9]),
+                ))
+            }
+        }
+        "sqlstatistics" => {
+            unsafe {
+                Ok(odbc_sys::SQLStatistics(
+                    statement.as_sys(),
+                    str_or_null(&function[1]),
+                    to_i16(&function[2]),
+                    str_or_null(&function[3]),
+                    to_i16(&function[4]),
+                    str_or_null(&function[5]),
+                    to_i16(&function[6]),
+                    to_i16(&function[7]),
+                    to_i16(&function[8]),
+                ))
+            }
+        }
+        "sqltableprivileges" => {
+            unsafe {
+                Ok(odbc_sys::SQLTablePrivileges(
+                    statement.as_sys(),
+                    str_or_null(&function[1]),
+                    to_i16(&function[2]),
+                    str_or_null(&function[3]),
+                    to_i16(&function[4]),
+                    str_or_null(&function[5]),
+                    to_i16(&function[6]),
+                ))
+            }
+        }
+         */
+        _ => Err(format!("unknown function {}", function_name)),
+    };
+
+    let sql_return_val = sql_return.unwrap();
+    if (sql_return_val != SqlReturn::SUCCESS) && (sql_return_val != SqlReturn::SUCCESS_WITH_INFO) {
```

Comment:
Do we expect SUCCESS_WITH_INFO at any point?
SUCCESS_WITH_INFO means something is not quite right.

Reply:
I changed this to only accept `SqlReturn::SUCCESS`.  I see now that `SUCCESS_WITH_INFO` is success with a warning message, that's a bit confusing.  I haven't seen a `SUCCESS_WITH_INFO` in my limited testing so I don't think we should expect it as success.